src/systems/log/LogPlayback.cc

```diff
@@ -299,6 +302,34 @@ bool LogPlaybackPrivate::Start(EntityComponentManager &_ecm)
     }
   }
 
+  msgs::LogPlaybackStatistics logStats;
```

Comment:
Could you add a check on `test/integration/log_system.cc` that verifies that the component was correctly created and has the correct values?

Reply:
39ca2b20e1f33e08d173513b83b827e692b73f82